demo/scss/_layout.scss

```diff
@@ -50,6 +50,14 @@ body {
     width: calc(100% - 200px);
   }
 
+  .bx--side-nav ~ & {
```

Comment:
sass nitpick: the `&` here is redundant right?

Reply:
@emyarod Tried to remove `&`, got different CSS output. Wanted to point out `~` existing just in case - Meaning, this is for `.container` which is a sibling of side nav.

Reply:
got it, thanks for clarifying @asudoh I overlooked that this block was nested